src/MonoMod.UnitTest/Core/Return2ByteStructIn32Bit.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MonoMod.Core;
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace MonoMod.UnitTest.Core
+{
+    [Obsolete("This test is replaced by DetouringMethodsReturningStructs")]
+    public class Return2ByteStructIn32Bit : TestBase
```

Comment:
If this is replaced, we don't need it at all, no?

Reply:
yes